src/ui/color/ColorSwatch.vue

```diff
@@ -20,11 +20,8 @@
  at runtime from the About dialog for additional information.
 -->
 <template>
-<div class="u-contents">
-    <div
-        v-if="canEdit"
-        class="grid-row"
-    >
+<div class="grid-row">
+    <template v-if="canEdit">
         <div
```

Comment:
can we add `v-if="canEdit"` here?

Reply:
Yeah  - https://v2.vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt